scripts/bv_get_morpho_deepsulci_models.py

```diff
@@ -64,7 +64,10 @@ def temporary(self, filetype):
 context = Context()
 
 destdir = options.output
-if not os.path.exists(destdir):
+if os.path.exists(destdir) and os.listdir(destdir):
+    context.write('skipping download:', destdir, 'exists and is not empty.')
+    sys.exit(0)
+else:
```

Comment:
this "else" case will happen if destdir exists and is empty, thus os.makedirs() will raise an error
```suggestion
elif not os.path.exists(destdir):
```